src/keycloak/chart/templates/uds-package.yaml

```diff
@@ -0,0 +1,84 @@
+apiVersion: uds.dev/v1alpha1
+kind: Package
+metadata:
+  name: keycloak
+  namespace: {{ .Release.Namespace }}
+spec:
+  network:
+    allow:
+      - description: "UDS Operator"
+        direction: Ingress
+        selector:
+          app.kubernetes.io/name: keycloak
+        remoteNamespace: pepr-system
+        remoteSelector:
+          app: pepr-uds-core-watcher
+        port: 8080
+        
+      # Temp workaround for any cluster pod
+      # @todo: remove this once cluster pods is a remote generated target
+      - description: "Keycloak backchannel access"
+        direction: Ingress
+        selector:
+          app.kubernetes.io/name: keycloak
+        remoteGenerate: Anywhere
+        port: 8080
+
```

Comment:
Missing a netpol here for "egress anywhere" to cover:
- todo for egress to postgres until we can lock that down further
- egress to OCSP once that is added

I can do this in the truststore PR, just thought it might be relevant here with postgres already?